addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -41,12 +48,32 @@ export class ProjectTaskFormController extends FormController {
         const versionedFieldName = 'description';
         const historyMetadata = record.data["html_field_history_metadata"]?.[versionedFieldName];
         if (!historyMetadata) {
+            this.notifications.add(
+                markup(
+                    `${escape(
+                        _t(
+                            "The task description lacks any past content that could be restored at the moment."
+                        )
+                    )}`
+                ),
```

Comment:
Did the linter do that? :sweat_smile: I suggest this instead:
```suggestion
                markup(escape(_t(
                    "The task description lacks any past content that could be restored at the moment."
                )))
```
Also, no need for a "fstring", right?

Reply:
Is there even a need to markup?

Reply:
Indeed, I think there is even no need to escape as we are not playing with user data nor special characters

Reply:
edit: I keep the escape as it seems to be used everywhere else, and it more robust in case of changes